models/gyroscope/ethereum/gyroscope_ethereum_trades.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(
+    schema = 'gyroscope_ethereum',
+    tags = ['dunesql'],
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "gyroscope",
+                                \'["fmarrr"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-04-20' %}
```

Comment:
No need for `{% set project_start_date = '2021-04-20' %}` as this is only important in the original `balancer.trades` spell.